packages/devextreme/testing/testcafe/tests/editors/overlays/toast.ts

```diff
@@ -0,0 +1,58 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { ClientFunction } from 'testcafe';
+import url from '../../../helpers/getPageUrl';
+import { testScreenshot } from '../../../helpers/themeUtils';
+import { insertStylesheetRulesToPage, setAttribute } from '../../../helpers/domUtils';
+
+fixture`Toast`
+  .page(url(__dirname, '../../container.html'));
+
+const types = ['info', 'warning', 'error', 'success'];
+const STACK_TONTAINER_SELECTOR = '.dx-toast-stack';
+
+const showToast = ClientFunction(
+  (type) => {
+    (window as any).DevExpress.ui.notify(
+      {
+        message: `Toast ${type}`,
+        type,
+        displayTime: 35000000,
+        animation: {
+          show: {
+            type: 'fade', duration: 0,
+          },
+          hide: { type: 'fade', duration: 0 },
+        },
+      },
+      {
+        position: 'top center',
+        direction: 'down-push',
+      },
+    );
+  },
+);
+
+const hideAllToasts = ClientFunction(() => {
+  (window as any).DevExpress.ui.hideToasts();
+});
+
+test('Toasts', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'Toasts.png', { element: STACK_TONTAINER_SELECTOR, shouldTestInCompact: true });
+  await testScreenshot(t, takeScreenshot, 'Toasts.png', {
+    element: STACK_TONTAINER_SELECTOR,
+    theme: process.env.theme?.replace('.light', '.dark'),
+  });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  await Promise.all(types.map((type) => showToast(type)));
+
+  await insertStylesheetRulesToPage(`${STACK_TONTAINER_SELECTOR} { padding: 20px; }`);
+  await setAttribute('body', 'class', `dx-theme-${process.env.theme}-typography`);
```

Comment:
I we will use `disablePageReloads` flag for this test we should think about leaving a blank page after passing the test. The better choose will add `dx-theme-${process.env.theme}-typography` class to stack container. Also, on your screenshot the background in dark theme is white, should be dark. It looks like the launch feature is not taken into account. I am going to help with it.